ca-mmis/X-hello-cammis-data
# FDP-4 Add index for greeting field and db migration

Add index for greeting field and db migration.  

Index was added using `db_index`: https://docs.djangoproject.com/en/2.2/ref/models/fields/#db-index

**How to test**
1. Build the Docker image: `sh docker/build-local.sh`
1. Launch a terminal/Docker instance: `sh docker/bash-local.sh`
1. Migrate the database: `python manage.py migrate`
1. Verify that the `data.0007_add_greeting_index` migration was applied successfully
